.eslintignore

```diff
@@ -1,4 +1,6 @@
 external
+node_modules
```

Comment:
We still want to lint `dist/jquery.js`, it did catch bugs in the compilation process in the past for me.